docs/content/python/guides/roles/sqlalchemy/_index.md

```diff
@@ -4,5 +4,8 @@ weight: 1
 description: >
     Use Oso with SQLAlchemy to handle all roles data management,
     expose roles to end users, and enforce authorization.
+aliases:
+ - ../../new-roles.html
```

Comment:
We link people to `/python/new-roles.html` in an error message.

Reply:
e.g. `/python/new-roles.html` redirects to `/python/guides/roles/sqlalchemy.html`